### DISCUSSION TITLE
hdfs 表函数 是否能支持读取压缩文件？

### DISCUSSION THREAD

**Discussion by xuweiit** [apache/doris#18140] [A - General / Q&A]
我在doris的官方文档上发现doris能直接读取hdfs的文件。由于我们的使用场景比较简单，直接读取hdfs也可以满足使用需求
`hdfs(
  "uri" = "..",
  "fs.defaultFS" = "...",
  "hadoop.username" = "...",
  "format" = "csv",
  "keyn" = "valuen" 
  ...
  );`
但是 使用之后发现，这个功能没有办法读取压缩文件，例如 gzip 或者lz4格式的文件。
在不把数据导入到BE的情况下，有什么好的解决办法吗？

**Comment by morningman** [comment]
It will be supported later
